src/golang/lib/workflow/engine/aq_engine.go

```diff
@@ -673,11 +672,9 @@ func (eng *aqEngine) execute(
 			}
 
 			// From here on we can assume that the operator has terminated.
-			if shouldPersistResults {
```

Comment:
should this depends on `isPreview` ?

Reply:
If I understand correctly, the value of `shouldPersistResults` is solely dependent on whether the request is a preview, so I replaced this with `isPreview`. @kenxu95 can you confirm?

Reply:
I might misunderstood - after reading more code in context, is the behavior now that we always persist in `execute()` and only clean-up in `preview`?

Reply:
I think so. In preview we never persist anything, but in `execute` (regular workflow execution) we do want to persist stuff.

Reply:
yeah they are the same thing, but I would like us to be consistent - either we always call it `isPreview` or always `shouldPersistResults`.